### PR TITLE
execdetails: make `ConcurrencyInfo` only appear once in explain analyze

### PR DESCRIPTION
**Pull request by xiongjiwei**
Concurrency will repeat many times when cte is support
image

### Release note 

- make ConcurrencyInfo only appear once in explain analyze

### CODE REVIEW

**Review by lzmhhh123 — APPROVED** [review]

File: util/execdetails/execdetails_test.go [hunk 1]
```diff
@@ -294,11 +294,8 @@ func TestRootRuntimeStats(t *testing.T) {
 	stmtStats.RegisterStats(pid, &RuntimeStatsWithCommit{
 		Commit: commitDetail,
 	})
-	concurrency = &RuntimeStatsWithConcurrencyInfo{}
-	concurrency.SetConcurrencyInfo(NewConcurrencyInfo("concurrent", 0))
-	stmtStats.RegisterStats(pid, concurrency)
 	stats := stmtStats.GetRootStats(1)
-	expect := "time:3s, loops:2, worker:15, concurrent:OFF, commit_txn: {prewrite:1s, get_commit_ts:1s, commit:1s, region_num:5, write_keys:3, write_byte:66, txn_retry:2}"
```
Comment (crazycs520) [inline util/execdetails/execdetails_test.go:301]: Should keep at least 1 `concurrent` information? avoid duplicate not means delete it.

Comment (xiongjiwei) [reply util/execdetails/execdetails_test.go:301]: the `worker:15` is the `concurrent` information, you can see it at L283

**Review by crazycs520 — APPROVED** [review]

**Pull request merged**